lib/rock_rms/response/content_channel.rb

```diff
@@ -0,0 +1,19 @@
+module RockRMS
+  module Response
+    class ContentChannel < Base
+      MAP = {
```

Comment:
`Id` is already included in `BASE_MAPPING` so we don't need to include it there.

I wonder if we could also move `ForeignKey` to `BASE_MAPPING` it might be on every Rock model.